posthog/feature_flags.py

```diff
@@ -131,18 +131,24 @@ def match_property(property, property_values) -> bool:
 
     if operator in ["is_date_before", "is_date_after"]:
         try:
-            parsed_date = parser.parse(value)
+            parsed_date = parser.parse(value, ignoretz=True)
```

Comment:
Hmmm I think this could be confusing because you're not restricted to setting date properties in a certain way on the app so you wouldn't expect a timezone you set to just be removed.

Reply:
Fair. We could make everything tz aware and default to utc if none known?